smart-frontend/app/src/components/uikit/LegendCard/LegendCard.module.scss

```diff
@@ -0,0 +1,62 @@
+:global {
+  body.theme-dark {
+    --icon-color: var(--color-grayUsual);
+    --icon-color-active: var(--color-dark1);
+    --text-color: var(--color-grayLighter);
+    --text-color-active: var(--color-ADQMblack);
+    --background-color: var(--color-newLight-50);
+    --background-color-active: var(--color-greenSaturated);
+  }
+  body.theme-light {
+    --icon-color: var(--color-ADCM);
+    --icon-color-active: var(--color-offWhite);
+    --text-color: var(--color-ADCM);
+    --text-color-active: var(--color-offWhite);
+    --background-color: var(--color-plateLight-70);
+    --background-color-active: var(--color-greenLogo);
+  }
+}
+
+.square {
```

Comment:
this class have local scope, but we named all classes with template: %componentName%, in this case `.legendCard {`

Reply:
Changed, thanks